packages/circularprogress/.storybook/main.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['@storybook/preset-typescript']
+}
```

Comment:
We might want to look into doing this in the top-level of the repo if it reduce friction to add types to future packages.